django_cookies_samesite/user_agent_checker.py

```diff
@@ -61,7 +61,10 @@ def is_uc_browser(self):
     def is_uc_browser_in_least_supported_version(self):
         major = int(self.user_agent.get("major") or "0")
         minor = int(self.user_agent.get("minor") or "0")
-        build = int(self.user_agent.get("patch") or "0")
+        try:
+            build = int(self.user_agent.get("patch") or "0")
```

Comment:
Hey @browniebroke,
First of all -> thank you for your contribution :bow:! Also, thanks are going to @abdul-rafiu for his fast response (that's a new thing for me to not worry about this repo when I get too busy :D).

I have a general concern:
This part of the code relies on release cycles/versions provided by browser vendors but to be defensive I would suggest making major/minor part of the version optional and add tests for that too.

But  (if that's not an urgent issue for you) I would wait for @abdul-rafiu ideas to fix them in one go.